src/scrabble.js

```diff
@@ -1,5 +1,47 @@
-function scrabble() {
+const scoreMap = {
+  A: 1,
+  E: 1,
+  I: 1,
+  O: 1,
+  U: 1,
+  L: 1,
+  N: 1,
+  R: 1,
+  S: 1,
+  T: 1,
+  D: 2,
+  G: 2,
+  B: 3,
+  C: 3,
+  M: 3,
+  P: 3,
+  F: 4,
+  H: 4,
+  V: 4,
+  W: 4,
+  Y: 4,
+  K: 5,
+  J: 8,
+  X: 8,
+  Q: 10,
+  Z: 10
+}
+
+function scrabble(anyword) {
+  if (anyword === null) {
```

Comment:
Alternatively, I think you could put "anyword !== 'string'" and cover more anomalies such as if the input was a number?. (not 100% sure, maybe note even 80% just trying to think outside the box 🤪)